gflownet.py

```diff
@@ -520,7 +524,7 @@ def forward_sample(self, envs, times, policy="model", model=None, temperature=1.
         t0_a_model = time.time()
         if policy == "model":
             with torch.no_grad():
-                action_logits = model(tf(states))
+                action_logits = model(tf(states))[..., : len(self.env.action_space) + 1]
```

Comment:
Did you use the three dots notation for a particular reason? Since it is just one dimension that we replace by the ellipsis, I would use colon notation to avoid confusion (`[:, : len(self.env.action_space) + 1]`